ug poemmmm

```diff
@@ -0,0 +1,3 @@
+for sale:
```

Comment:
So sad! Can you add the author (if it's known)?

Reply:
Ernest Hemmingway's 6 word poem!

Reply:
Ah! Can you add a line to the file to credit him?